scripts.py

```diff
@@ -2692,6 +2741,8 @@ def register_problem(cls, source, contribution, computed_sort_name, error_messag
 class Explain(IdentifierInputScript):
     """Explain everything known about a given work."""
 
+    name = "Explain everything known about a given work"
+
```

Comment:
We could also say that scripts which are only intended to be run from the command line should be `NoTimestampScripts`.

Reply:
There might be some that would be useful to keep track of, like a manual directory import, but I agree about this one.